### PR TITLE
BBTest tests pass on remote win and linux

### PR DESCRIPTION
**Pull request by ronenzaslaver**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -9,12 +9,13 @@ sphinx = "*"
...
 pytest-pythonpath = "*"
 pyvmomi = "*"
+pipenv = "*"
 
 [packages]
 dohq-artifactory = "*"
 psutil = "*"
 rpyc = "*"
-pywinrm = "*"
+pipenv = "*"
```
Comment (daonb) [inline Pipfile]: please remove `pipenv` dependency

File: README.md [hunk 2]
```diff
@@ -11,13 +11,25 @@ We've added a class property `LAB` that holds a dictionary defining the lab envi
...
+Fork and clone this repo.
+
+Testing
+-------
+In order to run all tests, you need to have python environment with all packages installed.
+Open a shell in the cloned repo folder:
 
 ```bash
 $ pipenv shell
+$ pipenv install
```
Comment (daonb) [inline README.md]: add `--dev`

File: bbtest/hosts.py [hunk 3]
```diff
@@ -33,41 +39,53 @@ def __init__(self, *args, **kwargs):
...
+        raise NotImplementedError('Missing method implementation')
 
     @property
     def bit(self):
-        raise NotImplementedError('Missing method implmentation')
+        return platform.machine()[-2:]
+
+    @property
+    def name(self):
+        return socket.gethostname()
```
Comment (daonb) [inline bbtest/hosts.py]: These two functions belong in `LocalHost`

File: cybereason/boxes/servers.py [hunk 4]
```diff
@@ -32,13 +32,13 @@ class PersonalizationBox(CRServerBox):
     Can run on any machine that can:
     Access the download server
     Untar the personalizer script
-    Run python 2.7
+    Run python 2.7 with packages 'protobuf' and 'google'
```
Comment (daonb) [inline cybereason/boxes/servers.py]: Please add a link to each of the packages mentioned

File: tests/test_os_agnostic.py [hunk 5]
```diff
@@ -13,44 +15,27 @@ def _test_operations(self):
...
-            'class': WindowsHost,
-            'boxes': [MyToDoBox],
-         },
-    }
-
-    address_book = {'host1': {'ip': '127.0.0.1',
-                              'auth': ('user', 'pass')},
-    }
+    @classmethod
+    def setUpClass(cls):
```
Comment (daonb) [inline tests/test_os_agnostic.py]: this belongs in BBTestCase

File: Pipfile (hunk 1 above)
Comment (ronenzaslaver) [reply Pipfile]: Added issue

File: README.md (hunk 2 above)
Comment (ronenzaslaver) [reply README.md]: Done

File: bbtest/hosts.py (hunk 3 above)
Comment (ronenzaslaver) [reply bbtest/hosts.py]: Done

File: cybereason/boxes/servers.py (hunk 4 above)
Comment (ronenzaslaver) [reply cybereason/boxes/servers.py]: Done

File: tests/test_os_agnostic.py (hunk 5 above)
Comment (ronenzaslaver) [reply tests/test_os_agnostic.py]: Done

**Pull request merged**